BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1517,8 +1525,7 @@ private void CheckMiss()
             if (!HasMissed && checkMiss)
             {
                 bool noProgress = MissileState == MissileStates.PostThrust && (Vector3.Dot(vessel.Velocity() - TargetVelocity, TargetPosition - vessel.transform.position) < 0);
-                bool pastGracePeriod = TimeIndex > ((vessel.LandedOrSplashed ? 0f : dropTime) + 180f / maxTurnRateDPS);
-                bool targetBehindMissile = !(MissileState != MissileStates.PostThrust && hasRCS) && Vector3.Dot(TargetPosition - transform.position, transform.forward) < 0f; // Allow thrusting RCS missiles to be behind the target
+                bool pastGracePeriod = TimeIndex > ((vessel.LandedOrSplashed ? 0f : dropTime) + Mathf.Clamp(maxTurnRateDPS / 15, 1, 8)); //180f / maxTurnRateDPS);
```

Comment:
Did the discussion on `180 / maxTurnRateDPS` vs `maxTurnRateDPS / 15` vs "only specific missiles can turn around" get resolved? It's not clear from the discussion in tea that it did.

Reply:
OK